client.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
1. Why not in tarantool/ directory?
2. We need to ensure the code works with python2 as well as python3.
3. flake8 reports some warings.